en/components/action-strip.md

```diff
@@ -94,12 +94,30 @@ This can be utilized via grid action components and we are providing two default
 
 >Note: When `IgxActionStripComponent` is a child component of the grid, hovering a row will automatically show the UI.
 
+### Row actions topic as part of the Grids section 
```

Comment:
Not needed to take the title so literally ;) Also, action-strip.md topic is not part of the Grids section - look where the grid topics are located and create a new one, where to put this content.